evm_loader/program/src/state_account.rs

```diff
@@ -13,13 +13,22 @@ use solana_program::{
     clock::Clock,
 };
 use std::cell::{RefMut, Ref};
+use std::collections::HashMap;
+use crate::account::TAG_ACCOUNT_V3;
 
+const ACCOUNT_CHUNK_LEN: usize = 1 + 1 + 32;
 
 pub enum Deposit<'a> {
     ReturnToOperator(Operator<'a>),
     Burn(Incinerator<'a>),
 }
 
+pub struct BlockedAccountMeta {
+    pub exists: bool,
+    pub is_writable: bool,
+}
+
+pub type BlockedAccounts = HashMap<Pubkey, BlockedAccountMeta>;
```

Comment:
You can simplify your code and convert `BlockedAccounts` to vector.

Reply:
Done.